.readthedocs.yaml

```diff
@@ -0,0 +1,28 @@
+# .readthedocs.yaml
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# Required
```

Comment:
I think you could make this "dependent" on the [distribute_built_dist.yml](https://github.com/viperleed/viperleed/blob/github-workflow-dev/.github/workflows/distribute_built_dist.yml) workflow that we will run automatically (after testing) every time we make a new release. I don't think it makes much sense to have documentation online for yet-to-be-released stuff.

Reply:
Sure, we can select the branch from which to build in the admin panel.

Reply:
Right now the action I have created builds only from master, which I'd always keep to date with the most recent version, but perhaps you're right, that we will eventually need other branches too